app/src/main/java/org/mozilla/fenix/share/SaveToPDFItem.kt

```diff
@@ -0,0 +1,69 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.share
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Divider
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * A save to PDF item.
+ * @param onClick event handler when the save to PDF item is clicked.
+ */
+@Composable
+fun SaveToPDFItem(
+    onClick: () -> Unit,
+) {
+    Row(
+        modifier = Modifier
+            .height(48.dp)
+            .fillMaxWidth()
+            .clickable(onClick = onClick),
+    ) {
+        Column {
+            Divider(
```

Comment:
🚗 nit: I'm wondering if it maybe makes more sense to to put this divider inline in the compose view in `ShareFragment`. This is just a thought for potential re-use: it seems like we may not want to assume a divider would be required anywhere this is used. It feels like the top-level UI would choose to use dividers to separate its child elements.

That said, I can see how that would be an awkward way to structure the code since its shared between XML and Compose. I don't have a strong opinion on what you choose to do, but thought it might be interesting food for thought.

Reply:
Similarly, if the `Divider` is re-located, the hierarchy could just be: 

```
Row(
   Icon()
   Text()
)

instead of a Row + Column + Row

Reply:
Yeah, I was wondering the same.  I'm not sure for this case if it's worth while trying to separated the divider from the UI, as there is not much to reuse 😅,  as we just have an icon and text, and it's customized to be part of the `ShareFragment` . I prefer to leave it as it's and wait until we can migrate the `ShareFragment` :)

Reply:
Sorry, @MozillaNoah I just refreshed and saw your message, thank for the suggestion!!

Reply:
Thanks both for the suggestions, I moved the divider to the xml as @MozillaNoah mention the compose UI would be much cleaner 😄.